### DISCUSSION TITLE
When trying to post to EDC url getting a Server Error 500

### DISCUSSION THREAD

**Discussion by anandt1201** [eclipse-edc/Connector#3279] [Q&A] [answered]
I am successfully able to callback post ProxyTransfer but i am facing issue when trying to post to the Url that received from the EDC. Does anyone have any idea why I am facing this issue 
![image](https://github.com/eclipse-edc/Connector/assets/30823144/b047f102-6757-49da-9a24-aed71c8f9fe5)

**Comment by wolf4ood** [comment]
Hi @anandt06 

do you have more info from the logs? 

Thanks

**Comment by anandt1201** [comment]
Hi @wolf4ood,
this is the log I am getting in control plane of both EDC's 
![image](https://github.com/eclipse-edc/Connector/assets/30823144/c8a0591c-b097-462a-8e7a-8be8b99125d9)

this is the log of control plane 
![image](https://github.com/eclipse-edc/Connector/assets/30823144/c6dd7a8d-f4ee-4601-a40b-9c98e2c8265d)


In my callback endpoint I am getting the call 8 times due to retry mechanism from EDC

Thanks

**Comment by wolf4ood** [comment]
@anandt06 

which version of EDC are you using?

**Reply by anandt1201** [reply]
0.3.3

**Reply by wolf4ood** [reply, marked as answer]
If you are referring to tractusx-edc 0.3.3 in case there are new releases available that you can try, link  [here](https://github.com/eclipse-tractusx/tractusx-edc). In any event i would advice to raise issues or discussion directly on the linked repo

Thanks

**Reply by anandt1201** [reply]
Ok Thank you I will do it
